fairreckitlib/dataloaders/config.ini

```diff
@@ -35,20 +34,8 @@ max_rating=419157.0
 timestamp=False
 
 [LFM-1B]
-# file_path=C:\Users\sanaz\Downloads\LFM-1b\LFM-1b_albums.txt.sample
-# headers=album-id,album-name,artist-id
-# file_path=C:\Users\sanaz\Downloads\LFM-1b\LFM-1b_artists.txt.sample
-# headers=artist-id,artist-name
-# file_path=C:\Users\sanaz\Downloads\LFM-1b\LFM-1b_tracks.txt.sample
-# headers=track-id,track-name,artist-id
-# file_path=C:\Users\sanaz\Downloads\LFM-1b\LFM-1b_LES.txt.sample
-# headers=user-id,artist-id,album-id,track-id,timestamp
-# file_path=C:\Users\sanaz\Downloads\LFM-1b\LFM-1b_users.txt.sample
-# headers=user-id,country,age,gender,playcount,registered_timestamp
-# file_path=C:\Users\sanaz\Downloads\LFM-1b\LFM-1b_users_additional.txt.sample
-# headers=user-id,novelty_artist_avg_month,novelty_artist_avg_6months,novelty_artist_avg_year,mainstreaminess_avg_month,mainstreaminess_avg_6months,mainstreaminess_avg_year,mainstreaminess_global,cnt_listeningevents,cnt_distinct_tracks,cnt_distinct_artists,cnt_listeningevents_per_week,relative_le_per_weekday1,relative_le_per_weekday2,relative_le_per_weekday3,relative_le_per_weekday4,relative_le_per_weekday5,relative_le_per_weekday6 relative_le_per_weekday7,relative_le_per_hour0,relative_le_per_hour1,relative_le_per_hour2,relative_le_per_hour3,relative_le_per_hour4,relative_le_per_hour5,relative_le_per_hour6,relative_le_per_hour7,relative_le_per_hour8,relative_le_per_hour9,relative_le_per_hour10,relative_le_per_hour11,relative_le_per_hour12,relative_le_per_hour13,relative_le_per_hour14,relative_le_per_hour15,relative_le_per_hour16,relative_le_per_hour17,relative_le_per_hour18,relative_le_per_hour19,relative_le_per_hour20,relative_le_per_hour21,relative_le_per_hour22,relative_le_per_hour23
-file_path=C:\Users\sanaz\Downloads\LFM-1b\LFM-1b_LES.mat
-headers=idx_users,idx_artists,LEs
+file_path=C:\Users\sanaz\Downloads\LFM-1b\
```

Comment:
maybe change this to a relative path along the lines of ..\Dataset\LFM-1b... so it can work for anyone instead of just you. a hardcoded filepath is not handy to have on the dev branch